content/docs/tools-ui/_index.md

```diff
@@ -10,17 +10,24 @@ guide_img: "/images/tools-ui/guide_img.png"
 primary: true
 ---
 
-Welcome to the **Open 3D Engine (O3DE)** Tools UI Developer's Guide! In this guide you will find guidelines around using the O3DE brand and UI style. Included is an introduction to component development and the custom Qt widget library for developers.
 
-If you are a looking to incorporate the O3DE logo into your project, please read the [Open 3D Branding Guidelines](/files/Open-3D-Branding-Guidelines-June-2021.pdf) PDF style guide to help stay aligned with our brand.
+Below you will find guidelines around using the **Open 3D Engine's (O3DE)** brand and UI style if you are looking for the O3DE logo to incorporate into your project, tools to develop the UI for your component development, or just make your life easier by checking out the custom Qt widget library for developers.
 
-You can also download a ZIP of our brand guidelines and logos: [O3DE Logo Pack.zip](/files/O3DE-Logo-Pack.zip)
+This page is constantly evolving with new updates for UI designs, patterns and assets that the contributors develop over time. This modular framework is called the **BlueJay Design System**. It contains all the components, patterns, and design elements that make up O3DE.
+
+If you have any questions regarding the BlueJay Design System or feel like something is incorrect or missing from it, you can [file an issue](https://github.com/o3de/sig-ui-ux/issues/new/choose) with the UI/UX Special Interest Group on GitHub.
+
+Feel like having a chat? Check out our #SIG-UX-UI channel on [Discord](https://discord.com/invite/xNb2q4SJKJ). The community is always happy to help!
 
 ## Tools UI Developer's Guide Contents
 
+![A banner of the BlueJay Design System](/images/tools-ui/bjds-banner.png)
+
+<br>
```

Comment:
I was having trouble following the opening paragraphs. One large suggestion for the intro. Move the banner to the top. Intro BJDS, enumerate the guide contents, provide links.

```suggestion
![A banner of the BlueJay Design System](/images/tools-ui/bjds-banner.png)

The **BlueJay Design System** is the modular UI framework of **Open 3D Engine (O3DE)**. This guide provides information for developers on the framework and how to use it when developing components, tools, and extensions for **O3DE Editor**.

This guide includes guidelines and information on the custom Qt widget library, the Qt Control Gallery tool, and a gallery of UI icon assets for O3DE. This guide also provides O3DE brand guidelines and O3DE brand assets that you can incorporate into your own projects.

If you have any questions regarding the BlueJay Design System or have a suggestion to improve it, you can [file an issue](https://github.com/o3de/sig-ui-ux/issues/new/choose) with the UI/UX Special Interest Group on GitHub.

Feel like having a chat? Check out our #SIG-UX-UI channel on [Discord](https://discord.com/invite/xNb2q4SJKJ). The community is always happy to help!

## Tools UI Developer's Guide Contents
```

Reply:
I like these recommendations team. I have forgotten the reason why the team suggested to have the banner moved down to the table. It may have been that @yuyihsu was concerned that the banner may deter developers in thinking that this page provides information to them and its not just a designer resource. (since this page for now aims towards editor extension developers)

but I okay with moving forward for this if that was your observation.

Reply:
I am resolving this with micronAMZN's suggestions.